ConnectionJustForToday/ConnectionJustForToday/Views/JFTReading/JFTReadingViewController.swift

```diff
@@ -7,12 +7,56 @@
 
 import UIKit
 
-class JFTReadingViewController: UIViewController {
-
+class JFTReadingViewController: UIViewController{
+    
+    
+    // MARK: - Outlets
+    
+    @IBOutlet weak var jftTitleLabel: UILabel!
+    @IBOutlet weak var jftDateLabel: UILabel!
+    @IBOutlet weak var jftQuoteLabel: UILabel!
+    @IBOutlet weak var jftAffirmationLabel: UILabel!
+    @IBOutlet weak var jftBodyTextView: UITextView!
+    @IBOutlet weak var jftCopyrightLabel: UILabel!
+    @IBOutlet weak var jftPostsTableView: UITableView!
+    
+    
+    // MARK: - Lifecycle
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        viewModel = JFTReadingViewModel(delegate: self)
     }
-
     
+    // MARK: - Properties
+    var viewModel: JFTReadingViewModel!
+    
+    // MARK: - Functions
+    
```

Comment:
Extra white space. Remove all extra white space beneath your marks. I won't comment on the rest of them